.github/workflows/pypi-package.yml

```diff
@@ -0,0 +1,26 @@
+name: Building & Publishing
+
+on:
+  push:
+    tags: 
+    - "v*"
+    - '!*dev*'
+    - '!*pre*'
+    - '!*post*'
```

Comment:
I _don't_ want the `publish-to-pypi` job to run for tags that are pushed with `dev` `pre` or `post` in the name. Just official releases.

Reply:
why not? these can be useful for testing and preparing a release. I would suggest that once this PR is merged we create a dev tag to check that this workflow is running.